### PR TITLE
Stats Docs

### PR DESCRIPTION
**Pull request by acpaquette**
Updates binning information in stats documentation

## Description
Stats uses a histogram to determine various statistics about the cube. This PR updates the stats documentation to better inform the user how the histogram is built for computing statistics.

## Related Issue
#3883

## Motivation and Context
More comprehensive explanation to the user on how the stats program computes various values.

## How Has This Been Tested?
N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/base/apps/stats/stats.xml [hunk 1]
```diff
@@ -11,6 +11,46 @@
...
+          <td>0 * cube multiplier + base</td>
+          <td>65535 * cube multiplier + base</td>
+          <td>65536</td>
+        </tr>
+        <tr border="1">
+          <td>Signed Word</td>
+          <td>-32768.0 * cube multiplier + base</td>
+          <td>32767.0 * cube multiplier + base</td>
+          <td>65536</td>
+        </tr>
```
Comment (scsides) [inline isis/src/base/apps/stats/stats.xml:43]: Even though they are the same, I suggest putting each on its own line. It's just easier to find

**Review by scsides — DISMISSED** [review]
This looks like what was asked for

File: isis/src/base/apps/stats/stats.xml (hunk 1 above)
Comment (acpaquette) [reply isis/src/base/apps/stats/stats.xml:43]: @scsides Each what exactly? This is how it looks on the docs page, I'm just not sure what you are referring to.
Screen Shot 2020-09-01 at 11 46 33 AM

Comment (acpaquette) [reply isis/src/base/apps/stats/stats.xml:43]: Test

**Review by scsides — COMMENTED** [review]
test

File: isis/src/base/apps/stats/stats.xml (hunk 1 above)
Comment (scsides) [reply isis/src/base/apps/stats/stats.xml:43]: The bottom row of the table combines "Unsigned Integer/Signed Integer/Real". I think it crowds the table and makes finding Real or Signed Integer difficult. My suggestion is to put each of these in their own row.

**Review by scsides — APPROVED** [review]
Looks good

**Pull request merged**